contracts/zero-ex/contracts/src/transformers/bridges/BaseGoerliBridgeAdapter.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+  Copyright 2023 ZeroEx Intl.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "./AbstractBridgeAdapter.sol";
+import "./BridgeProtocols.sol";
+import "./mixins/MixinUniswapV3.sol";
+import "./mixins/MixinUniswapV2.sol";
+
+contract BaseGoerliBridgeAdapter is
```

Comment:
I added [an escape patch](https://github.com/0xProject/protocol/commit/ec82c42c1b89ff79ba3133d3648b95b9f11415f8) for testnet couple weeks ago.

So we could potentially have a single adapter for both base and base goerli.

Reply:
or even just deploy `EthereumBridgeAdapter` on base goerli 🤔

Reply:
Do we know what sources of liquidity will be available on base mainnet at launch? Will it still be just UniV2 and UniV3?

Reply:
No, it's an unknown at this point. Really depends on CB's partnerships team results...

Reply:
It doesn't hurt to have extra sources in the bridge adapter 🤔 

It will actually save us from needing to deploy FQT again so we may want to start with a comprehensive list (include it if we think there is a good chance). 

How about starting with:
* Uniswap v2 
* Uniswap v3
* Solidly
* Kyber Elastic
* Balancer v2 batch
* Curve 
* Aave v3